library/geonetwork

```diff
@@ -19,10 +19,10 @@ Tags: 3.2.2-postgres, 3.2-postgres
 GitCommit: 7ba5f533d9ee31cf1a22d4d6a1a84fbb74f86466
 Directory: 3.2.2/postgres
 
-Tags: 3.4.0, 3.4, latest
-GitCommit: 067c56d23e1ea7a422e9883410b3db318b5f42b7
-Directory: 3.4.0
+Tags: 3.4.1, 3.4, latest
+GitCommit: 01e3e23e9b277940dcac471099408695c6d12ae3
+Directory: 3.4.1
 
-Tags: 3.4.0-postgres, 3.4-postgres, postgres
-GitCommit: 067c56d23e1ea7a422e9883410b3db318b5f42b7
-Directory: 3.4.0/postgres
+Tags: 3.4.1-postgres, 3.4-postgres, latest-postgres
```

Comment:
Whoops, not sure how we missed this before! :sweat:

https://github.com/docker-library/official-images/blob/master/README.md#tags-and-aliases

> In Docker, the `latest` tag is a special case, but it's a bit of a misnomer; `latest` really is the "default" tag. When one does `docker run xyz`, Docker interprets that to mean `docker run xyz:latest`. Given that background, no other tag ever contains the string `latest`, since it's not something users are expected or encouraged to actually type out (ie, `xyz:latest` should really be used as simply `xyz`). Put another way, having an alias for the "highest 2.2-series release of XYZ" should be `xyz:2.2`, not `xyz:2.2-latest`. Similarly, if there is an Alpine variant of `xyz:latest`, it should be aliased as `xyz:alpine`, not `xyz:alpine-latest` or `xyz:latest-alpine`.

Reply:
https://github.com/docker-library/official-images/pull/4143 :+1:

Reply:
@tianon it was not here before: I just added it in the latest commit, because I thought it was missing... let me remove it